src/card.css

```
@@ -0,0 +1,73 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+  
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+  
+.card__title {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+  
+.card__subtitle {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+  
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+  
+.card__price-block-name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+  
+.card__price-block-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+  
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  height: 40px;
+  width: 166px;
+  color: #fff;
+  background: #00acdc;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 700;
+  border: 1px solid rgba(0,172,220,0);
+  transition: 0.7s all ease;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.7s property1 ease, 0.7s property2 ease;
```